index.html

```diff
@@ -43,7 +43,141 @@ <h4 class="connectmywork"> LET’S CONNECT</h4>
       </div>
       </section>
 
-    </form>
+<section class="Workssection">
+    <div class="cardBack">
+    <a class="card1" ></a>
+            <h3  class="tonic" >Tonic</h3>
+         <div>
+            <ul class="works">
+                <li class="canopy"> CANOPY</li>
+                <li class="BackendDev">BackendDev</li>
+                <li class="num"> 2015 </li>
+            </ul>
+    <p class="section1"> A daily selection of privately personalized reads; no accounts or sign-ups required </p>
+</div>
+        <div >
+            <ul class="program1">
+                <li class="Program-skilhtml">html</li>
+                <li class="Program-skilhtml">css</li>
+                <li class="skiljava">javascript</li>
+            </ul>
+        </div>
+        <div class="buttonsee">
+            <button class="button1"> See Project</button>
+        </div>
```

Comment:
- [ ] Kindly  stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).as you can see the see project button is not the same as the orignal figma design the button is too bigger than the orignal design on figma kindly fix this to make your website look professsional .

![image](https://github.com/tes-nat/My_Portofolio/assets/92458236/b10c6755-078a-41c6-9ce9-5bca37a80110)

- [ ] your design
![Screenshot 2023-06-10 230359](https://github.com/tes-nat/My_Portofolio/assets/92458236/d02be708-e139-4d85-894e-b59fd75551ff)

Reply:
As suggested by the previous reviewer, kindly stick to the figma template as closely as possible. kindly work on the background color of the technologies and reduce the width of the button

your design :
![image](https://github.com/tes-nat/My_Portofolio/assets/20537712/2c31bb10-b4ce-4d36-9e8e-4ebed8b475a9)